tests/headingcommand.js

```diff
@@ -217,12 +204,12 @@ describe( 'HeadingCommand', () => {
 				);
```

Comment:
Perhaps we should be verifying such things in deltas constructors if it really matters?